mne/io/edf/edf.py

```diff
@@ -896,11 +902,10 @@ def _read_gdf_header(fname, stim_channel, exclude):
             meas_id['loc'] = loc
 
             date = np.fromfile(fid, np.uint64, 1)[0]
-            if date == 0:
-                date = datetime.datetime(1, 1, 1)
-            else:
+            if date != 0:
                 date = datetime.datetime(1, 1, 1) + \
```

Comment:
So you set `meas_date = DATE_NONE` in the top, and if something is read, then you overwrite it.

This snipped appears more than once maybe we can make a function:
```py
def _compute_edf_gdf_meas_date(date):
   if date == 0: 
       return DATE_NONE
   else:
      meas_date = datetime.datetime(1, 1, 1) + \
	    		   datetime.timedelta(date * pow(2, -32) - 367)
      meas_date (calendar.timegm(meas_date.utctimetuple()), 0)
      return meas_date
```

Reply:
I think a function for this is overkill